core/src/main/java/org/streamprocessor/core/transforms/DynamodbFn.java

```diff
@@ -110,14 +111,28 @@ public void processElement(@Element PubsubMessage received, OutputReceiver<Pubsu
                 out.output(
                         new PubsubMessage(payloadObject.toString().getBytes("UTF-8"), attributes));
             } catch (IllegalArgumentException e) {
-                LOG.error("IllegalArgumentException: ", e);
+                LOG.error(
+                        "exception[{}] step[{}] details[{}] entity[{}]",
+                        e.getClass().getName(),
+                        "DynamodbFn.processElement()",
+                        e.toString(),
+                        entity);
             } catch (org.json.JSONException e) {
-                LOG.error("org.json.JSONException: ", e);
+                LOG.error(
+                        "exception[{}] step[{}] details[{}] entity[{}]",
+                        e.getClass().getName(),
+                        "DynamodbFn.processElement()",
+                        e.toString(),
+                        entity);
             } catch (Exception e) {
-                LOG.error("Exception: ", e);
+                throw e;
```

Comment:
Do we need to catch this if we're just throwing it?

Reply:
I removed this part. Didn't know how the catch-scope would turn out so left the statement at first.

Reply:
It's still there?

Reply:
Ah scuzi, removed the extra one in the salesforce function but not in this dynamodb fn. Think I succeeded this time at least ^^